resources/certificate.rb

```diff
@@ -45,6 +45,11 @@
 property :ssl_endpoint, String,
   description: 'An SSL end-point from which to download the certificate'
 
+property :starttls, String,
+  default: '',
```

Comment:
Does this need to be an empty string or can it be the default which should be ``nil``?

Reply:
This has been fixed. Thanks again.